pkcs11/_pkcs11.pyx

```diff
@@ -225,7 +225,7 @@ class Slot(types.Slot):
 class Token(types.Token):
     """Extend Token with implementation."""
 
-    def open(self, rw=False, user_pin=None, so_pin=None):
+    def open(self, rw=False, user_pin=None, so_pin=None, p_auth=False):
```

Comment:
Let's use a meaningful parameter name, this is user facing API. `use_protected_auth` comes to mind. Can we also make sure it's generated into the API docs (in `types.py`).

Reply:
Will do.